cobalt/browser/cobalt_browser_interface_binders.cc

```diff
@@ -49,6 +51,10 @@ void PopulateCobaltFrameBinders(
   binder_map->Add<crash_annotator::mojom::CrashAnnotator>(
       base::BindRepeating(&crash_annotator::CrashAnnotatorImpl::Create));
 #endif  // BUILDFLAG(IS_ANDROIDTV)
+#if !BUILDFLAG(COBALT_IS_RELEASE_BUILD)
```

Comment:
A more Chromium way would be to use `#if defined(NDEBUG)` (see e.g. refs in [//base](https://source.chromium.org/search?q=NDEBUG&ss=chromium%2Fchromium%2Fsrc:base%2F&start=11), I couldn't find any documentation for it though). Is `COBALT_IS_RELEASE_BUILD` the preferred build?